rlbot_gui/gui/js/botmanager.js

```diff
@@ -0,0 +1,154 @@
+var globalJson
```

Comment:
I'm personally not a fan of globals in javascript.

Reply:
me too, but this one has a valid reason. the globaljson (filled at the beginning of the page load) gets reused without having to download and parse repofiles again.